scripts/ngi_pipeline_start.py

```diff
@@ -110,40 +121,69 @@ def __init__(self, *args, **kwargs):
+            help=("The path to one or more demultiplexed Illumina flowcell "
+                  "directories to process and analyze."))
     analyze_flowcell.add_argument("-p", "--project", dest="restrict_to_projects", action="append",
             help=("Restrict analysis to these projects. "
                   "Use flag multiple times for multiple projects."))
     # Add sub-subparser for project analysis
     ### TODO change to work with multiple projects
-    project_group = subparsers_analyze.add_parser('project', help='Start the analysis of a pre-parsed project.')
-    project_group.add_argument('analyze_project_dir', action='store', help='The path to the project folder to be analyzed.')
+    project_group = subparsers_analyze.add_parser('project',
+            help='Start the analysis of a pre-parsed project.')
+    project_group.add_argument('analyze_project_dir', action='store',
+            help='The path to the project folder to be analyzed.')
 
 
     args = parser.parse_args()
 
-    # The following option will be available only if the script has been called with the 'analyze' option
+    # These options are available only if the script has been called with the 'analyze' option
     if args.__dict__.get('restart_all_jobs'):
-        args.restart_failed_jobs = True
-        args.restart_finished_jobs = True
-        args.restart_running_jobs = True
+        if validate_dangerous_user_thing(action=("restart all FAILED, RUNNING, "
+                                                 "and FINISHED jobs, deleting "
+                                                 "previous analyses")):
+            args.restart_failed_jobs = True
+            args.restart_finished_jobs = True
+            args.restart_running_jobs = True
+    else:
+        if args.__dict__.get("restart_failed_jobs"):
+            args.restart_failed_jobs = \
+                validate_dangerous_user_thing(action=("restart FAILED jobs, deleting "
+                                                      "previous analysies files"))
+        if args.__dict__.get("restart_finished_jobs"):
+            args.restart_finished_jobs = \
+                validate_dangerous_user_thing(action=("restart FINISHED jobs, deleting "
+                                                      "previous analyseis files"))
+        if args.__dict__.get("restart_running_jobs"):
+            args.restart_finished_jobs = \
+                validate_dangerous_user_thing(action=("restart RUNNING jobs, deleting "
+                                                      "previous analysis files"))
+    # Charon-specific arguments ('organize' or 'analyze')
+    if args.__dict__.get("force_update"):
+        args.force_update = \
+                validate_dangerous_user_thing("overwrite existing data in Charon")
+    if args.__dict__.get("delete_existing"):
+        args.delete_existing = \
+                validate_dangerous_user_thing("delete existing data in Charon")
+
+
+    import ipdb; ipdb.set_trace()
```

Comment:
Well at least it reminds everyone to use `ipdb.set_trace()` and not just boring ol' `pdb`